### ISSUE TITLE
Predictions submit on website even after time passes

### ISSUE THREAD

**Issue by mriganksingh1** [coding-blocks/oneauth#660]
**Describe the bug**
ICC World Cup Predictions, we can make predictions even after submitting period is over.

**Expected behavior**
We should not be able to make predictions after predictions period is over by any way or any means.

**Logs**
In this, I have just changed my laptop time to 7:00 a.m after the match is over and i am able to make predictions.
![WhatsApp Image 2019-06-28 at 23 07 52](https://user-images.githubusercontent.com/44446528/60683584-1ed4a680-9eb6-11e9-8f2d-7b252e1fb981.jpeg)
![BOSS](https://user-images.githubusercontent.com/44446528/60683587-22682d80-9eb6-11e9-831f-4d52c1019dbf.jpeg)

**Comment by abhishek97** [comment]
Hi,
Yes the frontend UI shows incorrect state when you change time. However, you won't be able to submit prediction as there is a backend check in place to check this. 
Since this doesn't affect integrity of the system and doesn't affect most genuine users, this is taken as a low priority bug. Feel free to send in a PR with a fix ☺️

**Comment by mriganksingh1** [comment]
Contrarily i was able to submit predictions and also got free credits

**Comment by mriganksingh1** [comment]
I have reported this only after trying please check.

**Comment by mriganksingh1** [comment]
Ok now i see the check is placed but it was not there before when i have tried last time
